packages/site/src/ui/side-nav.js

```diff
@@ -53,6 +53,19 @@ const NAV_CONFIG = [
       { href: '/voice-tone', label: 'Voice & tone' }
     ]
   },
+  {
+    id: 'group-utils',
+    title: 'Utils',
+    hrefPrefix: '/utils',
+    items: [
+      { href: '/featureflags', label: 'Feature Flags' },
+      { href: '/icon', label: 'Icon Cli' },
+      { href: '/position', label: 'Position' },
+      { href: '/scrollable', label: 'Scrollable' },
+      { href: '/theme', label: 'Theme' },
+      { href: '/screenreaderonly', label: 'Screen Reader Only' }
+    ]
+  },
```

Comment:
cc @sweeting 

i move some components to a `utils` heading like we discussed. how are we feeling about this?

Reply:
@danethurber 👍 Definitely like them being grouped into a section.